src/components/OmouComponents/ReadMore.js

```diff
@@ -0,0 +1,31 @@
+import React,{useState} from 'react';
+import PropTypes from 'prop-types';
+import Typography from '@material-ui/core/Typography'
+import theme from "../../theme/muiTheme";
+
+const ReadMore = (props) => {
+
+    const [readMore, setReadMore] = useState(false);
+    const handleReadMoreClick = () => setReadMore(!readMore);
+
+    return (
+        <Typography variant="body1" align="left" style={{ wordBreak: "break-word" }}>
+            {
+                !readMore && props.body.length > 110 ? props.body.substring(0,60) + "..." : props.body
```

Comment:
change props.body to props.children

Reply:
make an explanatory variable for `props.body.length`. `const isOverTextLimit = props.body.length`

combine the shorten text with the bottom link
```
if(isOverTextLimit){
      return (<>
          isReadMoreClosed ? props.body.substring(0,60) + "..." : props.body
          // code for link toggle (lines 17 - 21)
      </>)
}
else {
     return <>{ReadMoreText}</>
}
```